### ISSUE TITLE
additions to the statistics page

### ISSUE THREAD

**Issue by bansp** [clarin-eric/standards#297]
As opposed to the discussion ticket #202 , this is a task description. 

https://standards.clarin.eu/sis/views/list-statistics.xq

Suggestions:
1. split the page into "Format statistics" and "Centre statistics"
2. under Format statistics, in the existing tables for Domains and Media types, make it possible to sort alphabetically in column 1 and numerically in column 2
3. under Centre statistics, add:
   1. table, with 4 columns: first: "RI", second: number of centres, third: number of depositing centres, fourth: number of curated centres;; and with 4 rows: all, CLARIN, Text+, DARIAH (these should preferably be dynamic, but we can leave the dynamicity for the future)
   2. I'll take care of the headings, a note warning that `all < CLARIN + T+ + DARIAH`, etc.

**Comment by bansp** [comment]
Adding priority as agreed at the meeting

**Comment by margaretha** [comment]
I change _Format Statistics_ to simply _Statistics_ and moved media types statistics to sanity check in [6ae4438](https://github.com/clarin-eric/standards/commit/6ae4438ba901c9caf5578d921f80533403f3a23a).

**Comment by bansp** [comment]
@bansp is going to take care of the descriptions / headers for the new pages

**Comment by bansp** [comment]
Looking at this now.

First thing: there is something odd in the "all" row in https://standards.clarin.eu/sis/views/list-statistics-centre.xq . I think I will just suppress it, at least for now.
I'll also add some prose, and maybe it would make sense to centre the numbers, formatted for two digits, ## -- except I can't be diving into the documentation to see how that should be coded.

**Comment by bansp** [comment]
In centre statistics, I felt something had to be done about the alignment, so I centred the numbers (no need to format them, it seems). Without that, the column divisions were not clear.
In general statistics, I originally wanted to align the numbers differently as well, but then I noticed that the flush-right comes from a module and decided not to investigate what would change elsewhere if I were to modify the text alignment. I just left it as it was.
One way or another, it seems to me that we can close this issue.
